docs/jobs.adoc

```diff
@@ -0,0 +1,89 @@
+// vim: tw=79
+= Jobs
+:toc:
+
+== List jobs
+
+List all jobs created.
+
+Sample Request
+
+----------
+curl -XGET
+http://127.0.0.1/api/1.0/jobs
+----------
+
+Sample Response
+
+----------
+Status: 200 OK
+[{
+	"integration_id": "717315a1-2a49-4cb7-826b-d2aa3bc4721a",
+	"status": "finished",
+	"flow": "ImportCluster",
+	"parameters": {
+		"integration_id": "717315a1-2a49-4cb7-826b-d2aa3bc4721a",
```

Comment:
Ok, I also overlooked that there is cluster_id in parameters section of response (on line 37). Just why is it different from integration_id if they are same?